pkg/parser/parser.go

```diff
@@ -184,7 +184,7 @@ func ValidateDeviceName(name string) error {
 	for _, c := range string(name[1 : len(name)-1]) {
 		switch {
 		case IsAlphaNumeric(c):
-		case c == '_' || c == '-' || c == '.' || c == ':':
+		case c == '_' || c == '-' || c == '.' || c == ':' || c == '+' || c == '=':
```

Comment:
Are we concerned with the fact that allowing an `=` could confict with the `vendor/class=name` pattern that we have for a fully-qualified CDI device name?

In addition, this was promted by the fact that we want to abuse the device name to pass base64-encoded data to something. These characters don't represent the set of valid base64 characters which includes `/`.

Note that this would also require a spec version bump.